tests/integration/test_charm.py

```diff
@@ -34,3 +41,107 @@ async def test_is_active(ops_test: OpsTest):
         raise_on_error=True,
         timeout=300,
     )
+
+
+def _safe_load_file_to_text(filename: str):
+    """Returns the contents of filename if it is an existing file, else it returns filename"""
+    try:
+        text = Path(filename).read_text()
+    except FileNotFoundError:
+        text = filename
+    return text
+
+
+def create_all_from_yaml(
```

Comment:
@ca-scribner @DnPlas could we use chisme for this instead? If not I think we desperately need that functionality in it.

Reply:
Yes I think @misohu you might have seen this in other charms?  We've got some shared code [here](https://github.com/canonical/charmed-kubeflow-chisme/tree/main/src/charmed_kubeflow_chisme/lightkube/batch) that should make this easier.  You'd use it in combination with `codes.load_all_yaml`

You might be the first person actually using it outside testing, so keep an eye out for something not working right

Reply:
Fixed